runtime/wasm/src/module/mod.rs

```diff
@@ -1677,8 +1677,7 @@ impl<C: Blockchain> WasmInstanceContext<C> {
         gas: &GasCounter,
         hash_ptr: AscPtr<AscString>,
     ) -> Result<AscPtr<AscString>, HostExportError> {
-        // Not enabled on the network, no gas consumed.
-        drop(gas);
+        // GasCounter is not enabled on the network, no gas consumed.
```

Comment:
Thank you for drawing my attention to this change. `drop(gas)` should go away indeed, but I think the previous comment was better!

Reply:
Indeed, will restore it!